api/children/childrenModel.js

```diff
@@ -47,7 +47,7 @@ const getEnrolledCourses = async (id) => {
 };
 
 const addEnrolledCourse = async (course) => {
-  // need to added middleware to the router that checks if the class exists because if it doesnt we get no response back from sever
+  // need to added middleware to the router that checks if the course exists because if it doesn't we get no response back from sever
```

Comment:
Very minor, but for the sake of consistency in quality here we could fix up the grammar from 'added' to 'add' and 'sever' to 'server'.